plugins/header_rewrite/ruleset.cc

```diff
@@ -51,7 +51,7 @@ RuleSet::add_condition(Parser &p, const char *filename)
     if (!c->set_hook(_hook)) {
       TSError("[%s] in %s: can't use this condition in hook=%d: %%{%s} with arg: %s", PLUGIN_NAME, filename, _hook,
```

Comment:
@zwoop, do you think it would make sense to print the hook name instead of the number? 
Having the number is a great help for the developers but the operators would not know what it means.